### DISCUSSION TITLE
Improve FAQ and Documentation

### DISCUSSION THREAD

**Discussion by ap0l0** [shine-jayakumar/insta-likecom-bot#100] [General]
Hi @shine-jayakumar 

What can be the best way to help you improving the documentation? Adding examples, screenshots, etc. to each of the functions. I think this will help new users and old ones navigate the script more easily.

**Comment by shine-jayakumar** [comment]
Good idea! Documentation can be improved. With time, more options and features get added. Although, currently the options have clear descriptions and more than a few examples for even new users to follow, however details sections for each feature can be added. 

Short instructional videos on topics like, installation on various platforms, usage of different features, could help.

Our community using the script, could share a short youtube video on whatever they can contribute, which can be compiled in an instructional video section.
